anagram/anagram.py

```diff
@@ -1,2 +1,20 @@
 def find_anagrams(word, candidates):
-    pass
+    for i in candidates:
+        count1 = 0
+        # print(i)
+        candsp = list(i)
+        # print(candsp)
+        for j in word:
+            # print(j)
+            if j in candsp:
+                candsp.remove(j)
+                # print(candsp)
+                count1 += 1
+            else:
+                #print("continue")
+                break
+        if len(word) == count1:
+            return(print(i))
```

Comment:
Instead of returning a print statement, we want to return a value. Specifically, we want to return a list of all the elements of candidate that are anagrams of word.

Reply:
It should work now. Lets test it